app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Admin::DashboardController < ApplicationController
   before_action :require_admin, only: [:index, :export]
+  before_action :set_default_locale, only: :index
```

Comment:
Will this work if I go directly to an admin page before I'm logged in that isn't the dashboard? It then directs to the login page and after logging in, directly back to the first url I tried.

Reply:
(@lucacug )